Formula/fceux.rb

```diff
@@ -1,43 +1,30 @@
 class Fceux < Formula
   desc "All-in-one NES/Famicom Emulator"
   homepage "https://fceux.com/"
-  url "https://downloads.sourceforge.net/project/fceultra/Source%20Code/2.2.3%20src/fceux-2.2.3.src.tar.gz"
-  sha256 "4be6dda9a347f941809a3c4a90d21815b502384adfdd596adaa7b2daf088823e"
-  revision 4
+  # version "2.3.0"
+  license "GPL-2.0-only"
+  revision 1
 
-  bottle do
```

Comment:
Please don't delete the `bottle` block.

Reply:
How is The bottle block handled? The previous binary builds are no longer valid

Reply:
It will be handled by @BrewTestBot. You just need to leave it where it is.

Reply:
Including the old sha sums?

Reply:
Just leave it untouched please.

Reply:
I just recovered it, should be good.